### ISSUE TITLE
Cannot be installed on win32 x64 despite having correct binaries

### ISSUE THREAD

**Issue by bdunn313** [dylanirlbeck/tailwind-ppx#96]
Even though windows binaries exist, installation fails on win32 x64.

Example output after running `yarn add --dev @dylanirlbeck/tailwind-ppx`

```bash
error C:\<redacted>\node_modules\@dylanirlbeck\tailwind-ppx: Command failed.
Exit code: 1
Command: node postinstall.js
Arguments:
Directory: C:\<redacted>\node_modules\@dylanirlbeck\tailwind-ppx
Output:
tailwind-ppx does not support this platform :(

If you want tailwind-ppx to support this platform natively,
please open an issue at our repository, linked above.
Specify that you are on the win32 platform,
and on the x64 architecture.
```

It appears that the `postinstall.js` script tries to find a platform-win32-x64 directory, but only a platform-windows-x64 directory exists. The script also does not properly check for the directory as the `supported` variable does not take into account the current dirname when trying to check to see if the architecture is supported.

**Comment by dylanirlbeck** [comment]
Hey @bdunn313! Thanks for raising this issue. My `postinstall.js` script was taken mostly from the [`spin` template](https://github.com/tmattio/spin-templates/blob/master/ppx/template/script/release-postinstall.js), so we'll be debugging/learning together 😄 I'll drop a review in your PR, I think the issue makes sense though.

**Comment by bdunn313** [comment]
@dylanirlbeck Thanks! I saw your comment and responded. I think we're good to go after your final spot-check/approval. Let me know if there's any follow up, and thanks for creating/maintaining this project! I'm looking forward to using it!

**Comment by dylanirlbeck** [comment]
@bdunn313 Just published version 0.7.8 on NPM. Fingers crossed that it works 🤞

**Comment by bdunn313** [comment]
@dylanirlbeck  Awesome! 🤞
